langchain/src/agents/executor.ts

```diff
@@ -299,6 +337,149 @@ export class AgentExecutor extends BaseChain<ChainValues, AgentExecutorOutput> {
     return getOutput(finish);
   }
 
+  async _takeNextStep(
```

Comment:
These are all only used in the streaming flow right?

Reply:
Correct